src/api/app/models/package.rb

```diff
@@ -876,9 +876,7 @@ def self.activity_algorithm
   end
 
   def activity
-    package = Package.find_by_sql("SELECT packages.*, #{Package.activity_algorithm} " \
-                                      "FROM `packages` WHERE id = #{id} LIMIT 1")
-    package.shift.activity_value.to_f
+    activity_index * 2.3276**((updated_at.to_f - Time.now.to_f) / 10_000_000)
```

Comment:
Can we get rid of ```self.activity_algorithm``` now?

Reply:
No because its used in https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/statistics_controller.rb#L90